plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
VP8 and VP9 may even belong to the same case, as they have the same number of arguments to the format string, and they only differ in `sdp_v_template_vp8` and `sdp_v_template_vp9`.